packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -79,6 +79,14 @@
     inline-size var(--calcite-animation-timing);
 }
 
+.container:focus {
+  @apply focus-inset z-default;
```

Comment:
Is there an alternative way to accomplish this? We want to avoid setting z-indices as much as possible. Headers have a designated z-index to use, which might be problematic here.

I think this might only be affecting Chrome too, so we could hold off a bit while we look into alternatives.

Reply:
I'm fine holding off to research this a bit more.

Even if we don't apply the z-index here, it might be nice to apply the consistent `focus-inset` styles.

Notes:

- In this scenario, we need to use an inset focus so that the focus is inside of the panel
- When the focus is inside of the panel, it can be overlapped by headers and other elements with a z-index
- A good solution would make sure the focus outline is above any slotted in content